src/webgl/interactions/ApplicationEvents.ts

```diff
@@ -0,0 +1,97 @@
+import { EventHandlers } from '../../types/api'
+import { EventSystem } from 'pixi.js'
+
+export default class ApplicationEvents {
```

Comment:
### ApplicationEvents
- similar to `RendererOptions`, this class will manage and update application events
- can also manage other properties like the `EventSystem` for easy access in relevant classes